indico/modules/events/timetable/controllers/manage.py

```diff
@@ -255,6 +260,52 @@ def _get_references(self, data: list[dict]) -> list[ContributionReference]:
         return references
 
 
+class RHTimetableScheduleContribution(RHManageTimetableBase):
+    session_management_level = SessionManagementLevel.coordinate
+
+    class ScheduleContribSchema(mm.Schema):
+        contrib_id = fields.Int(required=True)
+        start_dt = fields.DateTime(required=True)
+
+    @use_kwargs({
+        'block_id': fields.Int(load_default=None),
+    }, location='query')
+    def _process_args(self, block_id):
+        RHManageTimetableBase._process_args(self)
+        self.session_block = None
+        if block_id is not None:
+            self.session_block = self.event.get_session_block(request.view_args['block_id'])
+            if self.session_block is None:
+                raise NotFound
+            if self.session and self.session != self.session_block.session:
+                raise BadRequest
+
+    @use_kwargs({
+        'contribs': fields.List(fields.Nested(ScheduleContribSchema), required=True),
```

Comment:
So far we only use this endpoint to schedule a single contribution, but it accepts multiple contribs for when we add bulk scheduling